habitat/config/default.py

```diff
@@ -367,3 +376,31 @@ def get_config(
 
     config.freeze()
     return config
+
+
+def get_multi_task_config(
```

Comment:
I am wondering if this really warrants it's own function or just some flag to enable it? Or maybe we should just deprecate the single task format, thought @erikwijmans @mathfac ?

Reply:
Yes this could easily extend the current single-task format, tho I didn't want to introduce such a change without some feedback on the idea first.

Reply:
Yeah, I think this logic should just be in the normal config loader.  No need to even have a flag since config.TASKS can be the empty list and things will fall back to the old behavior.